codec/testdata/proto.proto

```diff
@@ -3,11 +3,13 @@ package cosmos_sdk.codec.v1;
 
 option go_package = "github.com/cosmos/cosmos-sdk/codec/testdata";
 
+// Dog defines a dog
```

Comment:
What does this change have to do with this PR?

Reply:
Idk. @jackzampolin changes?